helpers/control_node_setup.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Instructions to run this script. PLEASE READ!
+# 1. Update USERNAME, PASSWORD - only if RHEL control node is used
+# 2. Update RHEL_MAJOR_VERSION - as per the instructions given
+# 3. Make sure to run this script as root, in this project's root folder
+
+export USERNAME="changeme"
+export PASSWORD="changeme"
+
+if [[ $USERNAME = "changeme" && $PASSWORD = "changeme" ]]; then
+  echo "Please update the variables in the script before running it."
+  exit 1
+fi
+
+# RHEL_MAJOR_VERSION of the Control node should be as follows:
+# RHEL6: 6
+# RHEL7: 7
+# Fedora: Comment or remove the below line
+export RHEL_MAJOR_VERSION=7
```

Comment:
I completely missed updating this variable since it is farther down in the script. You can put this above the username and password check so its harder to miss. It would be worth it to add to the check as well, so users don't install the wrong EPEL version on their machines.